### ISSUE TITLE
Modules that are not found or have syntax errors may cause intern to hang

### ISSUE THREAD

**Issue by andyearnshaw** [andyearnshaw/intern-systemjs-loader#2]
This didn't seem to be a problem during development for Chrome, but was always an issue with PhantomJS.  It's possible that a new version of Chrome exhibits this behaviour now.

There might be something we can do in the loader part to make sure errors are properly thrown.

**Comment by andyearnshaw** [comment]
From what I can tell, the 404 error (module not found) is logged to the console, which triggers a POST to `/__intern/`.  The (formatted) request body looks like this:

``` javascript
[{
    "sequence": 0,
    "sessionId": "d6c5df2198267ef5f75424ce6d3089c4",
    "payload": [
        "fatalError",
        {
            "name": "Error",
            "message": "(SystemJS) XHR error (404 Not Found) loading http://localhost:9000/boo\n\tError: XHR error (404 Not Found) loading http://localhost:9000/boo\n\tError loading http://localhost:9000/boo as \"boo\" from http://localhost:9000/tests/sample.spec.js",
            "stack": "(SystemJS) XHR error (404 Not Found) loading http://localhost:9000/boo\n\tError: XHR error (404 Not Found) loading http://localhost:9000/boo\n\tError loading http://localhost:9000/boo as \"boo\" from http://localhost:9000/tests/sample.spec.js",
            "sessionId": "d6c5df2198267ef5f75424ce6d3089c4"
        }
    ]
}]
```

The error thrown at the server is:

```
Error: Repeated sequence for session 0411dc217a6eeb3564422d7ed266c810: 0 last 0 cur
    at Proxy._publishInSequence (file:///Volumes/HOME/Users/andyearnshaw/Development/intern-systemjs-loader/node_modules/intern/lib/Proxy.js:189:11)
    at file:///Volumes/HOME/Users/andyearnshaw/Development/intern-systemjs-loader/node_modules/intern/lib/Proxy.js:62:20
    at Array.map (native)
    at IncomingMessage.<anonymous> (file:///Volumes/HOME/Users/andyearnshaw/Development/intern-systemjs-loader/node_modules/intern/lib/Proxy.js:61:56)
    at emitNone (events.js:86:13)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:926:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

I'm leaning towards a bug in Intern here, so will post upstream.

**Comment by andyearnshaw** [comment]
Fixed in intern, 3.4 release.
